wordpress/wp-content/plugins/cds-base/classes/Modules/Contact/ContactForm.php

```diff
@@ -16,6 +16,25 @@ public function register()
         add_shortcode('contact-form', [$this, 'render']);
     }
 
+    public function checkBox($name, $id, $value): void
+    {
+        ?>
+        <div class="gc-input-checkbox">
+            <input
+                    name="<?php echo $name; ?>"
+                    class="gc-input-checkbox__input"
+                    id="<?php echo $id; ?>"
```

Comment:
Ooh, this breaks web conventions [according to MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id):

> id's value must not contain [whitespace](https://developer.mozilla.org/en-US/docs/Glossary/Whitespace) (spaces, tabs etc.). Browsers treat non-conforming IDs that contain whitespace as if the whitespace is part of the ID. In contrast to the [class](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#attr-class) attribute, which allows space-separated values, elements can only have one single ID value.

Ours are just full sentences.

<img width="853" alt="Screen Shot 2022-03-01 at 16 04 55" src="https://user-images.githubusercontent.com/2454380/156248712-43ff3984-63e0-4765-acc9-aea26b89798b.png">

We could strip all the whitespace and we would still get unique ids.